### DISCUSSION TITLE
can't use temp file as a payloads in Nuclei version3.0

### DISCUSSION THREAD

**Discussion by Alaminhossyin** [projectdiscovery/nuclei#4261] [Q&A]
The previous version works fine! That's it!

**Comment by ehsandeep** [comment]
@Alaminhossyin, it's from update in [nuclei v2.9.9](https://github.com/projectdiscovery/nuclei/releases/tag/v2.9.9), disabled the default ability to read files from any location on the system due to potential security risks. You can optionally enable the same behavior using ﻿`-lfa` option.

```
   -lfa, -allow-local-file-access        allows file (payload) access anywhere on the system
```

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
